ROADMAP.md

```diff
@@ -0,0 +1,55 @@
+# cellxgene roadmap
+
+We are very exited for _cellxgene_ to become a valuable tool in collaborations
+between computational biologists and experimental biologists working on
+single-cell transcriptomics data. _cellxgene_ is in active developmentis in
+active development, and we've love to include the community as we plan new
+features to work on.
+
+In the future, we plan to improve cellxgene in the ways described below.
+
+Please note: this roadmap may change at any time.
```

Comment:
```suggestion
Please note: this roadmap is subject to change.
```
It sounds a little less volatile which makes the roadmap seem a bit more useful.